src/java/org/apache/cassandra/net/InboundMessageHandler.java

```diff
@@ -302,10 +302,7 @@ protected void fatalExceptionCaught(Throwable cause)
 
         JVMStabilityInspector.inspectThrowable(cause);
 
-        if (cause instanceof Message.InvalidLegacyProtocolMagic)
-            logger.error("{} invalid, unrecoverable CRC mismatch detected while reading messages - closing the connection", id());
```

Comment:
@ilyazr what is wrong with keeping everyhing as is and just use "noSpam1m" here?